schemas/src/registry_schemas/schemas/incorporation.json

```diff
@@ -0,0 +1,37 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://bcrs.gov.bc.ca/.well_known/schemas/incorporation",
+    "required": [
+        "incorporation"
+    ],
+    "type": "object",
+    "title": "Incorporation Filing",
+    "properties": {
+        "incorporation": {
```

Comment:
This should be incorporationApplication, to match header.name options. That has been the approach so far and enables searching for json properties based on filing.name in our code.

Reply:
ok great, will update!

Reply:
Yep just did a simple check of the syntax and it works! FYI you can use https://www.jsonschemavalidator.net to check syntax with simplified examples.